src/session/session_api.c

```diff
@@ -1465,6 +1513,16 @@ __wt_session_range_truncate(
         WT_TRET(start->reset(start));
     if (stop != NULL)
         WT_TRET(stop->reset(stop));
+
+#ifdef HAVE_DIAGNOSTIC
+    if (is_truncate) {
```

Comment:
I don't see a need for this `is_truncate` variable. The cursors are initialized with NULL, so checking them against NULL before closing should be enough.

Reply:
Thanks, I've removed this.